.env.example

```
@@ -0,0 +1,9 @@
+# Copy this file to .env and replace the placeholders with local-only values.
+# Never commit the generated .env file.
+SNOWTHING_DB_USERNAME=snowuser
+SNOWTHING_DB_PASSWORD=replace-with-a-local-password
+SNOWTHING_DB_ROOT_PASSWORD=replace-with-a-different-root-password
+
+# Optional credentials for CommentCreateTest's fixed snowthing_test MySQL schema.
+SNOWTHING_TEST_DB_USERNAME=snowuser
+SNOWTHING_TEST_DB_PASSWORD=replace-with-a-local-test-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SNOWTHING_TEST_DB_PASSWORD key should go before the SNOWTHING_TEST_DB_USERNAME key

(UnorderedKey)